### PR TITLE
add issue template for bug and docs update

### PR DESCRIPTION
**Pull request by frankiefab100**
This fixes issue #32

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Nextcloud (please complete the following information):**
```
Comment (briangershon) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: I don't know much about Nextcloud, is this from a previous project? I think Nextcloud could be removed.

File: .github/ISSUE_TEMPLATE/docs_update.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (briangershon) [inline .github/ISSUE_TEMPLATE/docs_update.md:1]: On second thought, let's remove the `docs_update.md` form and just stick with one bug report template. We can always beef up the form later if needed.

**Review by briangershon — CHANGES_REQUESTED** [review]
Thank you @frankiefab100 for your template contribution!

I just had a couple of feedback items. Let me know what you think.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (frankiefab100) [reply .github/ISSUE_TEMPLATE/bug_report.md:27]: Yes, it is. I will remove it.

File: .github/ISSUE_TEMPLATE/docs_update.md (hunk 2 above)
Comment (frankiefab100) [reply .github/ISSUE_TEMPLATE/docs_update.md:1]: Alright, I will be making the necessary changes.

**Comment by briangershon** [comment]
Looks good, thanks @frankiefab100! Merging...

**Pull request merged**